### DISCUSSION TITLE
Linux kernel upgrade

### DISCUSSION THREAD

**Discussion by jpaali** [polarfire-soc/polarfire-soc-documentation#313] [Linux build systems for PolarFire SoC] [answered]
Hello,
Do you have any plans to take a newer Linux kernel version into use?
At least there is a linux-6.1-mchp branch in linux4microchip repository.

**Comment by vfalanis** [comment, marked as answer]
Hi @jpaali ,

Yes, this is planned. We will transition to the Linux4Microchip 6.1 kernel in our next official PolarFire SoC Linux release. If you follow/watch our repositories or the GitHub discussions, you should get a notification each time we do a release :)

Thanks
